en/docs/learn/adding-and-configuring-a-service-provider.md

```diff
@@ -73,6 +73,15 @@ the following:
     Name** is a required field.  
     ![sp-name](../assets/img/using-wso2-identity-server/sp-name.png)
 
+    !!! note "Configure Service Provider Name Validation"
+           The default javascript regex used to validate the service provider name entered in the Management Console is `^[a-zA-Z0-9\\s.+_-]*$`. 
+           This regex can be modified by adding the following configuration to the `deployment.toml` file located in `<IS_HOME>/repository/conf/`.
+
+        ``` java
+        [service_provider]
+        sp_name_java_script_regex = '<required_javascript_regex>'
+        ```          
```

Comment:
This needs to be indented to the same level as the note content

Reply:
this configuration details should come inside the note.